eventemitter2.d.ts

```diff
@@ -11,12 +11,12 @@ declare module eventemitter2 {
          * @default '.'
          * @description the delimiter used to segment namespaces.
          */
-        delimiter?: string, 
```

Comment:
Sorry, I use visual studio code, it should have automatically removed the whitespaces.